components/eam/src/physics/cam/cam_diagnostics.F90

```diff
@@ -602,27 +606,34 @@ subroutine diag_init()
 
    ! outfld calls in diag_surf
 
-   call addfld ('SHFLX',horiz_only,    'A','W/m2','Surface sensible heat flux')
-   call addfld ('LHFLX',horiz_only,    'A','W/m2','Surface latent heat flux')
-   call addfld ('QFLX',horiz_only,    'A','kg/m2/s','Surface water flux')
+   call addfld ('SHFLX',horiz_only,    'A','W/m2','Surface sensible heat flux', &
+   standard_name='surface_upward_sensible_heat_flux')
+   call addfld ('LHFLX',horiz_only,    'A','W/m2','Surface latent heat flux', &
+   standard_name = 'surface_upward_latent_heat_flux')
+   call addfld ('QFLX',horiz_only,    'A','kg/m2/s','Surface water flux', &
+   standard_name = 'water_evapotranspiration_flux')
 
    call addfld ('TAUX',horiz_only,    'A','N/m2','Zonal surface stress')
    call addfld ('TAUY',horiz_only,    'A','N/m2','Meridional surface stress')
-   call addfld ('TREFHT',horiz_only,    'A','K','Reference height temperature')
+   call addfld ('TREFHT',horiz_only,    'A','K','Reference height temperature', &
+      standard_name='air_temperature')
    call addfld ('TREFHTMN',horiz_only,    'M','K','Minimum reference height temperature over output period')
    call addfld ('TREFHTMX',horiz_only,    'X','K','Maximum reference height temperature over output period')
-   call addfld ('QREFHT',horiz_only,    'A','kg/kg','Reference height humidity')
-   call addfld ('U10',horiz_only,    'A','m/s','10m wind speed')
-   call addfld ('RHREFHT',horiz_only,    'A','fraction','Reference height relative humidity')
+   call addfld ('QREFHT',horiz_only,    'A','kg/kg','Reference height humidity', &
+    standard_name = 'specific_humidity')
```

Comment:
This one does have a more suitable standard name, "surface_specific_humidity".
See also https://cfconventions.org/Data/cf-standard-names/28/build/cf-standard-name-table.html

Reply:
I used what is in the published CMIP6 data.  I think reference height is not the surface  so we can't call this surface_specific_humidity.

Reply:
Ok, then, Rob. This field is the closest we can have for 'surface' specific humidity, though.

Reply:
I looked into how to better handle the issue of reference height and related variables that are defined, e.g., at a single height (or time, or location). The `standard_name` should be, as it is in this PR, the same as the full 3D field. However, there is a CF-ism called a _scalar coordinate variable_ that can be used to define the (in this case) reference height. That scalar coordinate would then be added to the coordinates attribute of the data variable (`RHREFHT` or `TREFHT` etc). It seems too intrusive to add new scalar coordinates this late in the game for v2, though maybe this could be done for v3. See examples here: http://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/cf-conventions.html#scalar-coordinate-variables